app/_hub/kong-inc/openid-connect/overview/_index.md

```diff
@@ -1750,17 +1750,17 @@ Nice, as you can see the plugin even added the `X-Consumer-Id` and `X-Consumer-U
 
 One of the main vulnerabilities of OAuth are bearer tokens because presenting a valid bearer token is enough proof to access a resource. This can create problems since the client presenting the token isn't validated as the legitimate user that the token was issued to. 
 
-Certificate-bound access tokens can solve this problem by binding tokens to clients. This ensures the legitimacy of the token because the it requires proof that the sender is authorized to use a particular token to access protected resources. 
+Certificate-bound access tokens can solve this problem by binding tokens to clients. This ensures the legitimacy of the token because it requires proof that the sender is authorized to use a particular token to access protected resources. 
 
-To use enable certificate-bound access for OpenID Connect, you must ensure that the auth server is set up to generate OAuth 2.0 Mutual TLS Certificate Bound Access Tokens.
-If you are configuring this in Keycloak, see the previous [Keycloak configuration](#keycloak-configuration) section.
+To enable certificate-bound access for OpenID Connect, it is necessary to ensure that the auth server is set up to generate OAuth 2.0 Mutual TLS Certificate Bound Access Tokens.
```

Comment:
Though it may be obvious to folks who know this well, not everyone may be aware of the way mtls works and behind the scene setup needed to ensure all of this works. We can hint at it without being verbose.  So I was thinking a mention of the setup such as - **CA should be trusted by both Authz server and Kong**  might be helpful.

Reply:
I was trying to offload as much as possible of the setup to the external docs of the auth server, but I think a quick mention like the one you suggest can be a good addition.